analyzers/src/SonarAnalyzer.CSharp/Rules/LoopsAndLinq.cs

```diff
@@ -0,0 +1,137 @@
+﻿/*
```

Comment:
we should have a ticket to create a code fixer for this one

Reply:
Added here: https://github.com/SonarSource/sonar-dotnet/issues/4894